tracing/trace_state.hh

```diff
@@ -677,6 +678,12 @@ inline void add_table_name(const trace_state_ptr& p, const sstring& ks_name, con
     }
 }
 
+inline void add_alternator_table_name(const trace_state_ptr& p, const sstring& cf_name) {
```

Comment:
The implementation makes me wonder if we shouldn't just call it add_table_name (without the word "alternator") - exactly like the name of the method you call.
The parameter could be called full_table_name instead of "cf_name" - again like in the method that you call.
And as I mentioned above it should take std::string_view, not a sstring reference.